v22.2/alter-backup-schedule.md

```diff
@@ -37,7 +37,7 @@ Parameter | Description
 `collectionURI` | The URI where you want to store the backup. See [Backup file URLs](backup.html#backup-file-urls) for detail on forming the URI.
 `option` | Control the backup behavior with a comma-separated list of these [options](#backup-options).
 `RECURRING crontab` | Specify when the backup should be taken. By default, these are incremental backups. A separate schedule may be created automatically to write full backups at a regular cadence, depending on the frequency of the incremental backups. You can likewise modify this separate schedule with `ALTER BACKUP SCHEDULE`. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron). All times in UTC. <br><br>Example: `'@daily'` (run daily at midnight)
-`FULL BACKUP crontab / ALWAYS` | Specify when to take a new full backup. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron) or as `ALWAYS`. <br><br>`FULL BACKUP ALWAYS` will trigger `RECURRING` to always take full backups. **Note:** If you do not have an Enterprise license then `ALWAYS` is the only accepted value of `FULL BACKUP`. <br><br>If you omit the `FULL BACKUP` clause, the default backup schedule will be as follows: <ul><li>If `RECURRING` <= 1 hour: Default to `FULL BACKUP '@daily'`</li><li>If `RECURRING` <= 1 day: Default to `FULL BACKUP '@weekly'`</li><li>Otherwise: Default to `FULL BACKUP ALWAYS`</li></ul>
+`FULL BACKUP crontab / ALWAYS` | Specify when to take a new full backup. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron) or as `ALWAYS`. <br><br>`FULL BACKUP ALWAYS` will trigger `RECURRING` to always take full backups. **Note:** If you do not have an Enterprise license then `ALWAYS` is the only accepted value of `FULL BACKUP`. If `ALWAYS` is not specified, users will receive a warning.<br><br>If you omit the `FULL BACKUP` clause, the default backup schedule will be as follows: <ul><li>If `RECURRING` <= 1 hour: Default to `FULL BACKUP '@daily'`</li><li>If `RECURRING` <= 1 day: Default to `FULL BACKUP '@weekly'`</li><li>Otherwise: Default to `FULL BACKUP ALWAYS`</li></ul>
```

Comment:
I'm honestly not sure if we need to mention the warning here as well, since the warning itself also tells the user what to do, i.e., include `FULL BACKUP ALWAYS` to silence it and that only full backups will run. 

Perhaps you could edit this and the other instances on the `CREATE SCHEDULE FOR BACKUP` page, to something like:

"**Note:** If you do not have an Enterprise license then you can only take full back ups, therefore `ALWAYS` is the only accepted value of `FULL BACKUP`.

Reply:
ditto. Ideally we separate out info  meant only for core users from info for fall users.